oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -749,9 +749,8 @@ static int mca_spml_ucx_ctx_create_common(long options, mca_spml_ucx_ctx_t **ucx
 
 int mca_spml_ucx_ctx_create(long options, shmem_ctx_t *ctx)
 {
-    mca_spml_ucx_ctx_t *ucx_ctx = NULL;
+    mca_spml_ucx_ctx_t *ucx_ctx          = NULL;
     mca_spml_ucx_ctx_array_t *idle_array = &mca_spml_ucx.idle_array;
-    mca_spml_ucx_ctx_array_t *active_array = &mca_spml_ucx.active_array;
```

Comment:
maybe use `active_array` in this func to be consistent with `idle_array `?

Reply:
done